.styleci.yml

```diff
@@ -1,10 +1,7 @@
-preset: psr2
+preset: psr12
 
 finder:
   not-name:
     - MethodWithHHVMReturnType.php
     - MockingParameterAndReturnTypesTest.php
     - SemiReservedWordsAsMethods.php
-
-disabled:
-  - braces

```

Comment:
PSR-12 now allows single line style Closures, so we don't need to disable this any more.

Reply:
:tada: woohoo